lm_eval/tasks/jsquad.py

```diff
@@ -33,11 +33,11 @@
 
 
 DYNAMIC_MAX_LENGTH = os.getenv("DYNAMIC_MAX_LENGTH", "true").lower()
+JASQUAD_METRIC = datasets.load_metric(jasquad.__file__)
```

Comment:
I think this is not desirable in lm-eval-harness perspective. Because the metric is loaded even if JSQuAD/JaQuAD are not used. 
In fact, SQuAD in lm-eval-harness does it.  
https://github.com/EleutherAI/lm-evaluation-harness/blob/master/lm_eval/tasks/squad.py#L35